py/selenium/deprecated.py

```diff
@@ -0,0 +1,31 @@
+from warnings import warn
+
+removed_from_version = "4.12"
```

Comment:
I would rather we provided a \`version=\` flag maybe, what do you think? We are not great and getting deprecated things out atm, I think due to also not following semantic versioning makes it a tad harder (imo), but allowing us to define this vs hard coding would be useful

Reply:
I have changed it to `version` now. The reason I had named it as `removed_from_version` is that, it should not be confused with current version of selenium. It is the version (future) from which something will be removed.